pkg/services/folder/folderimpl/store.go

```diff
@@ -18,9 +18,6 @@ type store interface {
 	// Use Move to change a dashboard's parent ID.
 	Update(ctx context.Context, cmd *folder.UpdateFolderCommand) (*folder.Folder, error)
 
-	// Move changes the given folder's parent folder uid and applies any necessary permissions changes.
-	Move(ctx context.Context, cmd *folder.MoveFolderCommand) (*folder.Folder, error)
```

Comment:
Don't we still need `Move` in the interface?

Reply:
`Move` needs to remain in the service interface, but it's "just" a fancy update as far as the store is concerned. 
I imagine we'll make a new service -> store update command struct that includes all the information the store needs